src/tiles/item-image.ts

```diff
@@ -38,201 +29,41 @@ export class ItemImage extends LitElement {
 
   render() {
     return html`
-      <div class=${ifDefined(this.imageBoxClass)}>
-        ${this.model?.mediatype === 'audio'
-          ? this.waveformTemplate
-          : this.backgroundImageTemplate}
-        ${this.tileActionTemplate}
-      </div>
+      ${this.isWithWaveformMediatype
+        ? this.waveformImageTemplate
+        : this.backgroundImageTemplate}
     `;
   }
 
   private get imageSrc() {
     return `${this.baseImageUrl}/services/img/${this.model?.identifier}`;
   }
 
-  // Templates
-  private get backgroundImageTemplate() {
-    return html`
-      <div
-        class=${this.imageClass}
-        style="background-image:url(${this.imageSrc})"
-      ></div>
-    `;
+  private get isWithWaveformMediatype() {
+    return (
+      this.model?.mediatype === 'audio' || this.model?.mediatype === 'etree'
```

Comment:
wow, TIL `etree` is a mediatype

Reply:
Q: some `etree` / lam items have primary images. what shows up on their tile? waveform bg w/ primary image?

Reply:
were currently just detecting if the loaded image is a waveform based on the aspectRation... so if we detected it's a waveform, then we'll load the waveform with background, but if it has an actual image cover/album cover of some sort then we'll load that image instead